src/frontend-samples/issues-sample/IssuesClient.ts

```diff
@@ -0,0 +1,1317 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/* eslint-disable */
+/* tslint:disable */
```

Comment:
Maybe add a comment here that this file was auto-generated. Similar language to the README.

Reply:
Added back in the auto generated header